bin/clean_os_dupl.sh

```diff
@@ -4,7 +4,7 @@
 set -e
 
 # Installing necessary tool for the script: python-glanceclient
-sudo pip install --upgrade python-glanceclient
+sudo pip install python-openstackclient==3.11.0
```

Comment:
Eventually I've decided to switch once and for all to the recommend `openstack-cli` instead of old `glance` client.